### PR TITLE
Feature/apmw 196 add coordinates to room

### LINKED ISSUE #196: add coordinates to rooms
As a user I want to be able to navigate to specific rooms. 
Therefore the room needs coordinates as well.

### PR DESCRIPTION
**Pull request by Max784**
closes #196

### CODE REVIEW

**Comment by Tratori** [comment]
Can the Locateable module be applied here, too (https://github.com/hpi-swt2/compass-portal-blue/pull/147) ?
@joh-dah @Max784

**Comment by taminob** [comment]
> Can the Locateable module be applied here, too (#147) ? @joh-dah @Max784

Yes, I think it should be included in the room model. Simply adding `include Locatable` should do it and fix the currently missing verifications.

File: false [hunk 1]
```diff
@@ -0,0 +1,2 @@
+      invoke  active_record
+      create    db/migrate/20220114102847_add_admin_to_users.rb
```
Comment (taminob) [inline false:2]: should be removed

File: spec/views/locations/new.html.erb_spec.rb [hunk 2]
```diff
@@ -25,8 +25,6 @@
       assert_select "input[name=?]", "location[location_latitude]"
 
       assert_select "input[name=?]", "location[location_longitude]"
-
-      assert_select "input[name=?]", "location[location_longitude]"
```
Comment (taminob) [inline spec/views/locations/new.html.erb_spec.rb:29]: good catch!

**Review by taminob — APPROVED** [review]
We (@cambryx and me) pushed a commit to fix the failing test. Otherwise fine!

**Pull request merged**